src/simplification/constant_gate_reducer.hpp

```diff
@@ -235,6 +252,17 @@ class ConstantGateReducer_ : public ITransformer<CircuitT>
         return gate_id;
     }
 
+    /**
+     * Create a subcircuit whose response will be the output (created instead of the outputs
+     * already existing in the original circuit, the value of which became known after the transformation)
```

Comment:
"""
Create a gadget circuit in a BENCH basis which value is always const. This gadget may be used to replace irreducible CONST_* gates (e.g. when const gate is an output of a circuit).
"""